src/backend/README.md

```diff
@@ -1,26 +1,108 @@
-The backend is clio's view into the database. The database could be either PostgreSQL or Cassandra.
```

Comment:
Unless I'm reading this wrong, I think this file is being reverted back to a previous state. Let's fix that.

Reply:
So, the state of my branch is correct, and looks like its the same as `README.md` at `master`.  However, I have no idea why its showing these as changes...

Reply:
This is master in your fork: https://github.com/natenichols/xrpl-reporting/blob/master/src/backend/README.md
This is the rpc-refactor branch: https://github.com/natenichols/xrpl-reporting/blob/rpc-refactor/src/backend/README.md